models/yang/openconfig-ztp.yang

```diff
@@ -0,0 +1,86 @@
+module openconfig-ztp {
```

Comment:
Are we allowed to use the `openconfig` prefix for our custom Yang models? I thought we should be using separate SONiC Yang models

Reply:
Transformer does not support Sonic yangs that do not have one-to-one mapping with Redis Db. So an "Openconfig style yang" is used